frontend/lib/laletterbuilder/letter-builder/my-letters.tsx

```diff
@@ -22,12 +26,145 @@ export const LaLetterBuilderMyLetters: React.FC<ProgressStepProps> = (
+          <button
+            className="button is-primary is-medium jf-is-next-button"
+            onClick={downloadPdf}
+          >
+            <Trans>Download letter</Trans>
+          </button>
+        )}
+      </div>
+      <hr />
+      <Accordion question={"What's next?"} questionClassName="">
+        <h2>
+          <Trans>Allow 14 days for a response</Trans>
+        </h2>
+        <p>
+          <Trans>
+            If your landlord or property manager doesn’t respond, you should
+            file a complaint.
+          </Trans>
+        </p>
+        <span>
+          <Trans>For LA City residents</Trans>
+        </span>
+        <p>
+          <Trans>Call LAHD at (866) 557-7368</Trans>
+        </p>
+
+        <span>
+          <Trans>For LA county residents</Trans>
+        </span>
+        <p>
+          <Trans>Call LADBS at (213) 473-3231</Trans>
+        </p>
+        <h2>
+          <Trans>Mark your calendar</Trans>
+        </h2>
+        <p>
+          <Trans>
+            Make sure you keep your schedule clear during the requested access
+            dates:
+          </Trans>
+        </p>
+      </Accordion>
+    </div>
+  );
+};
+
 const MyLettersContent: React.FC = (props) => {
+  const { session } = useContext(AppContext);
+  const processedLetters = session.habitabilityLetters?.filter(
+    (el) => !!el.fullyProcessedAt && !el.letterSentAt
+  );
+  const sentLetters = session.habitabilityLetters?.filter(
+    (el) => !!el.letterSentAt
+  );
+
   return (
     <div className="jf-my-letters">
-      <p className="subtitle">See all your finished and unfinished letters</p>
-
+      <p className="subtitle">
+        <Trans>See all your finished and unfinished letters</Trans>
+      </p>
       <CreateOrContinueLetter />
+      {processedLetters?.map((el, i) => (
+        <CompletedLetterCard
+          key={`processed-letter-${i}`}
+          pdfBytes={el.pdfBase64}
+        >
+          <h3>
+            <Trans>JustFix is preparing your letter</Trans>
+          </h3>
+          <p>
+            <Trans>
+              Your tracking number will appear here once the letter has been
+              sent.
+            </Trans>
+          </p>
+        </CompletedLetterCard>
+      ))}
+      {sentLetters?.map((el, i) => {
+        const sentDate = new Date(el.letterSentAt!);
+        const dateString = sentDate.toLocaleDateString("en-US", {
+          day: "numeric",
+          month: "short",
+          year: "numeric",
+        });
+        return (
+          <CompletedLetterCard key={`sent-letter-${i}`} pdfBytes={el.pdfBase64}>
+            <h3>{`${li18n._(
+              t`JustFix sent your letter on`
+            )} ${dateString} `}</h3>
+            <p className="jf-laletterbuilder-letter-tracking">
+              <Trans>USPS tracking number:</Trans>{" "}
+              <OutboundLink
+                href={`https://tools.usps.com/go/TrackConfirmAction_input?strOrigTrackNum=${el.trackingNumber}`}
```

Comment:
I recommend using `USPS_TRACKING_URL_PREFIX` here, or if that's not quite the right URL, add it to `loc.json` (or rename that file to `usps_tracking.json` since that's all it does) and refer to that constant.

I think EvictionFree and NoRent both rely on that constant, e.g.:
```
{props.wasMailedToLandlord && (
        <p>
          <Trans>
            A hard copy of your form has also been mailed to your landlord via
            USPS mail. You can also track the delivery of your hard copy form
            using USPS Tracking:
          </Trans>{" "}
          <a
            data-jf-show-href-only-in-plaintext
            href={USPS_TRACKING_URL_PREFIX + props.trackingNumber}
          >
            {props.trackingNumber}
          </a>
        </p>
      )}
```